frontend/src/routes/FacilityBooking/ViewConflicts/ConflictBookingModal.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { ConfirmationModal } from '../../../components/ConfirmationModal'
+import { handleCreateBooking } from '../../../store/facilityBooking/action'
+import { Booking } from '../../../store/facilityBooking/types'
+
+export default function ConflictBookingModal(props: { booking: Booking | undefined }) {
+  const [open, setOpen] = useState(false)
+  /* TODO implement function to toggle state to true */
+  function changeStatus() {
+    setOpen(!prevOpen)
+  }
+  function onRightClick() {
+    /*TODO to update createbooking function in store instead*/
```

Comment:
is update to make the additional call to book? kiv prince's pr..

Reply:
Yes.

Reply:
remove comment